src/backend/src/modules/pia-intake/metadata/pia-status.metadata.ts

```diff
@@ -283,8 +283,63 @@ export const piaStatusMetadata: Partial<
           },
         ],
       },
+      [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+        allow: true,
+      },
+    },
+  },
+  [PiaIntakeStatusEnum.PENDING_COMPLETION]: {
+    updates: {
+      allow: false,
+    },
+    transition: {
+      [PiaIntakeStatusEnum.INCOMPLETE]: {
+        allow: true,
+        conditions: [
+          {
+            accessType: [UserTypesEnum.CPO],
```

Comment:
Is pending completion applicable to Delegate PIA? If not, can we add piaType=STANDARD condition as well?

<applicable to all transitions>

Reply:
I'm pretty sure that, in either case, Pending Completion is a valid status before Complete. In a Delegated flow, a CPO would still have to upload the PIA to the PID. Until they do so, the PIA would be 'Pending completion."